volume-group-operator/controllers/persistentvolumeclaim/settings.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package persistentvolumeclaim
+
+import (
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+)
+
+var (
+	persistentVolumeClaim = "PersistentVolumeClaim"
+	pvcPredicate          = predicate.Funcs{
```

Comment:
what is this for?

Reply:
that the controller will only watch update and create events, we don't need a delete event, because if a user wants to delete a PVC that is already in a `volumeGroup` he needs to remove it from the `volumeGroup`, which is an update event